package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "xhtml-purifier",
-	"version": "0.3.1",
+	"version": "0.3.2",
```

Comment:
in this case, pretty much nobody but us uses this, so it's not a huge deal. but it's good practice to keep on semver as we work on our open source foo.

(and BTW - somebody actually forked this repo and used it as a base for a Meteor HTML sanitizer. so this code is in use outside of Zaption!)